Assets/Scripts/Miscellaneous/TurnManager.cs

```diff
@@ -55,6 +55,11 @@ public void ChangeTurn()
 
     void Update()
     {
+        if (leftWizardController.IsActive)
```

Comment:
Czy te rzeczy nie powinny być ustawiane na starcie ? Przy każdym update'cie będziemy ustawiać wartości z template'u ? Nie będzie nam to zerowało parametrów Health, Condition, ManaPoints etc. ?